architecture/3-DataStructures/6-InterfaceListItem.yaml

```diff
@@ -0,0 +1,23 @@
+name: Component List Item
+purpose: Identifies an interface on a part or tool.
+template: |
+  id: {{id}}
+  description: {{description}}
+  descriptionLong: {{descriptionLong}}
+elements:
+  - key: id
+    type: string
+    itemType: ''
+    description: Part's ID (or part number).  Format = single word, only lowercase letters, and may contain hyphens and underscores.
```

Comment:
`Part's ID (or part number)` should be `Interface ID`

Reply:
Resolved [4cbcc20](https://github.com/ODU-CGA-CubeSat/dof-cubesat/pull/2/commits/4cbcc2047b672037edb01d55d017f9938b041063)